ci/desktop/.buildkite/pipeline-prs.yml

```diff
@@ -0,0 +1,15 @@
+steps:
+  - label: ":pipeline: Run Trinity Desktop (PRs) pipeline"
+    command:
+      - "yarn"
+      - "cd src/shared && yarn && cd ../.."
```

Comment:
I would suggest creating npm scripts for these. Cleaner that way.

Reply:
Good idea 👍